source/common/http/http2/codec_impl.cc

```diff
@@ -781,7 +781,10 @@ Status ConnectionImpl::onBeforeFrameReceived(const nghttp2_frame_hd* hd) {
   ENVOY_CONN_LOG(trace, "about to recv frame type={}, flags={}, stream_id={}", connection_,
                  static_cast<uint64_t>(hd->type), static_cast<uint64_t>(hd->flags), hd->stream_id);
   current_stream_id_ = hd->stream_id;
-
+  StreamImpl* stream = getStream(hd->stream_id);
+  if (stream != nullptr) {
+    stream->updateReceivedBytes(hd->length + 9);
```

Comment:
can you refactor have the 9 be a constant somewhere with a more readable name? Thanks

Reply:
Actually I am not sure about the meaning of 9 here, will ask Yan later.

Reply:
It's the size of the frame header, which in the standard is 9 octets.   The nghttp2 frame_header struct excludes this from length